### ISSUE TITLE
LDWEF allows form_completed to be set in the backend when the form is not complete

### ISSUE THREAD

**Issue by pbchase** [ctsit/linear_data_entry_workflow#34]
LDWEF allows form_completed to be set in the backend when the form is not complete.  This example shows such an event from the REDCap Event Log:

![form_completed_updated_early](https://user-images.githubusercontent.com/3356493/40258067-55a91160-5abe-11e8-96e7-95f2dfddfd41.png)

Can this be fixed?

**Comment by ChemiKyle** [comment]
This appears to be the same as issue #43 which was addressed.

**Comment by ChemiKyle** [comment]
Hey @jrpence can you check up on if this is still an issue?

**Comment by jrpence** [comment]
It is possible to import data that marks forms as complete, there doesn't seem to be a way to do it through the form.

**Comment by pbchase** [comment]
Yes. The form complete flag can be set by setting <form_name>_complete = 2 for the record_id(s) and redcap_event_name(s) you want to mark as complete.
